homeassistant/components/sharkiq/services.yaml

```diff
@@ -0,0 +1,15 @@
+clean_room:
+  target:
+    entity:
+      integration: "sharkiq"
+      domain: "vacuum"
+
+  fields:
+    rooms:
+      required: true
+      advanced: true
```

Comment:
The purpose of the `advanced` flag is to hide service fields from users which have not enabled the "advanced" flag in their user profile. I don't think that's useful here since the `rooms` service field is not optional?

Reply:
Thanks for the detailed explanation here.  I think since I updated services.yaml the commit checks yelled at me because my new "clean_rooms" service did not have an icon defined.  I created and icons.json file and specified an icon for it and that seems to have done the trick.